app/sidekiq/my_first_job.rb

```diff
@@ -0,0 +1,7 @@
+class MyFirstJob
+  include Sidekiq::Job
+
+  def perform()
+  
```

Comment:
what's the functionality of this method

Reply:
I forget to remove this method. now i will remove.